xilem/examples/mason.rs

```diff
@@ -29,9 +32,30 @@ fn app_logic(data: &mut AppData) -> impl WidgetView<AppData> {
         Axis::Vertical
     };
 
-    let sequence = (0..count)
-        .map(|x| button(format!("+{x}"), move |data: &mut AppData| data.count += x))
+    let flex_sequence = (0..count)
+        .map(|x| {
+            (
+                button(format!("+{x}"), move |data: &mut AppData| data.count += x),
+                if data.active {
+                    FlexSpacer::Flex(x as f64)
+                } else {
+                    FlexSpacer::Fixed((count - x) as f64)
+                },
+            )
+        })
         .collect::<Vec<_>>();
+
+    let flexy_flex_sequence = [2, 3, 4].map(|c| {
```

Comment:
I guess this is our catch-all playground example? This also uses `[impl ViewSequence; N]` btw.

Reply:
Yeah - it's fine to stick anything in here :laughing: